### PR TITLE
flush_endpoint capability added if flush_enabled (fixes #496)

### PR DESCRIPTION
**Pull request by lavish205**

### CODE REVIEW

**Comment by Natim** [comment]
Thanks you @lavish205 for this. I will let @leplatrem review it :+1:

File: kinto/__init__.py [hunk 1]
```diff
@@ -67,7 +67,16 @@ def main(global_config, config=None, **settings):
     # Scan Kinto views.
     kwargs = {}
     flush_enabled = asbool(settings.get('flush_endpoint_enabled'))
-    if not flush_enabled:
+
+    if flush_enabled:
+        config.add_api_capability(
+            "flush_endpoint",
```
Comment (leplatrem) [inline kinto/__init__.py]: nitpick: I wonder if we should name it `flush` only (?)

Comment (Natim) [inline kinto/__init__.py]: With @almet we though of ``flush``, ``flush_endpoint``, ``flush_endpoint_enabled``.
``flush_endpoint`` seemed more explicit that just ``flush``

In the future we should define a way to phrase:

- The kinto server has got the capability to have ``attachments``
- The kinto server has got the capability to have ``schema``
- The kinto server has got the capability to have ``changes``
- The kinto server has got the capability to have ``flush_endpoint``

Comment (leplatrem) [inline kinto/__init__.py]: Makes sense.

**Pull request merged**